### PR TITLE
Update Slack Link to signup page

### PR DESCRIPTION
**Pull request by itdependsnetworks**
# Closes: N/A
# What's Changed

Simple signup link documentation change.

# TODO
- [ ] Explanation of Change(s)

### CODE REVIEW

**Comment by itdependsnetworks** [comment]
I do not think the current link makes sense. The user will not have anything to do from that link, since there is a chicken & egg problem.

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Nautobot runs as a web application atop the [Django](https://www.djangoproject.c
 
 The complete documentation for Nautobot can be found at [Read the Docs](https://nautobot.readthedocs.io/en/stable/).
 
-Questions? Comments? Start by perusing our [GitHub discussions](https://github.com/nautobot/nautobot/discussions) for the topic you have in mind, or join us in the **#nautobot** Slack channel on [Network to Code](https://networktocode.slack.com)!
+Questions? Comments? Start by perusing our [GitHub discussions](https://github.com/nautobot/nautobot/discussions) for the topic you have in mind, or <a href="https://slack.networktocode.com/">join the **#nautobot**</a> Slack channel on Network to Code community!
```
Comment (bryanculver) [inline README.md:14]: This is a markdown file, not HTML.

File: nautobot/docs/docker/index.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Nautobot Docker Images
 
-Nautobot is packaged as a Docker image for use in a production environment. The published image is based on the `python:3.7-slim` image to maintain the most compatibility with Nautobot deployments. The Docker image and deployment strategies are being actively developed, check back here or join the **#nautobot** Slack channel on [Network to Code](https://networktocode.slack.com) for the most up to date information.
+Nautobot is packaged as a Docker image for use in a production environment. The published image is based on the `python:3.7-slim` image to maintain the most compatibility with Nautobot deployments. The Docker image and deployment strategies are being actively developed, check back here or <a href="https://slack.networktocode.com/">join the **#nautobot**</a> Slack channel on the Network to Code community for the most up to date information.
```
Comment (bryanculver) [inline nautobot/docs/docker/index.md:3]: Please change link to Markdown format.

**Review by bryanculver — CHANGES_REQUESTED** [review]
Please fix link formats.

File: nautobot/core/templates/500.html [hunk 3]
```diff
@@ -36,7 +36,7 @@
 Python version: {{ python_version }}
 Nautobot version: {{ nautobot_version }}</pre>
                         <p>
-                            If further assistance is required, please post to the <a href="https://networktocode.slack.com/">#nautobot channel on Network to Code's Slack</a>.
+                            If further assistance is required, please <a href="https://slack.networktocode.com/">join the **#nautobot**</a> Slack #nautobot channel on Network to Code's Slack community and post your question.
```
Comment (bryanculver) [inline nautobot/core/templates/500.html:39]:
```suggestion
                            If further assistance is required, please <a href="https://slack.networktocode.com/">join the <b>#nautobot</b></a> channel on Network to Code's Slack community and post your question.
```

File: README.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ Nautobot runs as a web application atop the [Django](https://www.djangoproject.c
 
 The complete documentation for Nautobot can be found at [Read the Docs](https://nautobot.readthedocs.io/en/stable/).
 
-Questions? Comments? Start by perusing our [GitHub discussions](https://github.com/nautobot/nautobot/discussions) for the topic you have in mind, or join us in the **#nautobot** Slack channel on [Network to Code](https://networktocode.slack.com)!
+Questions? Comments? Start by perusing our [GitHub discussions](https://github.com/nautobot/nautobot/discussions) for the topic you have in mind, or [join the **#nautobot**](https://slack.networktocode.com/) Slack channel on Network to Code community!
```
Comment (bryanculver) [inline README.md:14]:
```suggestion
Questions? Comments? Start by perusing our [GitHub discussions](https://github.com/nautobot/nautobot/discussions) for the topic you have in mind, or [join the **#nautobot**](https://slack.networktocode.com/) channel on Network to Code Slack community!
```

File: nautobot/docs/docker/index.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # Nautobot Docker Images
 
-Nautobot is packaged as a Docker image for use in a production environment. The published image is based on the `python:3.7-slim` image to maintain the most compatibility with Nautobot deployments. The Docker image and deployment strategies are being actively developed, check back here or join the **#nautobot** Slack channel on [Network to Code](https://networktocode.slack.com) for the most up to date information.
+Nautobot is packaged as a Docker image for use in a production environment. The published image is based on the `python:3.7-slim` image to maintain the most compatibility with Nautobot deployments. The Docker image and deployment strategies are being actively developed, check back here or [join the **#nautobot**](https://slack.networktocode.com/) Slack channel on the Network to Code community for the most up to date information.
```
Comment (bryanculver) [inline nautobot/docs/docker/index.md:3]:
```suggestion
Nautobot is packaged as a Docker image for use in a production environment. The published image is based on the `python:3.7-slim` image to maintain the most compatibility with Nautobot deployments. The Docker image and deployment strategies are being actively developed, check back here or [join the **#nautobot**](https://slack.networktocode.com/) channel on the Network to Code Slack community for the most up to date information.
```

**Comment by bryanculver** [comment]
Suggested the same call to action on all three spots:

> ...join the [#nautobot](https://slack.networktocode.com/) channel on Network to Code's Slack community...

I think the link should still be on "Network to Code's Slack community" and not "#nautobot" because they have to join the community first before joining the channel. Thoughts?

**Pull request merged**